### DISCUSSION TITLE
Modifying handle1 of holdable component with StatusEffect doesn't work

### DISCUSSION THREAD

**Discussion by BiffW** [FakeFishGames/Barotrauma#14759] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Modifying properties of holdable component with StatusEffect like railgun shell does. Holdpos and holdangle are updated correctly but handle1 is not.

### Reproduction steps

1. hold a railgun shell and go into water
2. see hand position relative to item origin not being updated

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
